File: internal/content/testdata/bench/rust/nc-sum-two-integers.rs
fn get_sum(a: i64, b: i64) -> i64 {
    let mask: i64 = 0xFFFFFFFF;
    let mut a = a & mask;
    let mut b = b & mask;
    while (b & mask) != 0 {
        let carry = ((a & b) << 1) & mask;
        a = (a ^ b) & mask;
        b = carry;
    }
    a &= mask;
    if a <= 0x7FFFFFFF {
        a
    } else {
        !(a ^ mask)
    }
}

File: internal/content/testdata/bench/go/nc-sum-two-integers.go
func get_sum(a, b int) int {
	mask := 0xFFFFFFFF
	for b&mask != 0 {
		carry := ((a & b) << 1) & mask
		a = (a ^ b) & mask
		b = carry
	}
	a &= mask
	if a <= 0x7FFFFFFF {
		return a
	}
	return ^(a ^ mask)
}
